examples/source_examples/microsoft_intune/connector.py

```diff
@@ -0,0 +1,126 @@
+# This is a simple example for how to work with the fivetran_connector_sdk module.
+# The code will retrieve managed Devices from Microsoft InTune and create one table called managed_devices
+# You will need to provide your own Microsoft credentials for this to work --> "tenant_id", "client_id", and "client_secret" in configuration.json
+# Relevant Microsoft API documentation: https://learn.microsoft.com/en-us/graph/api/intune-devices-manageddevice-list?view=graph-rest-1.0
+# See the Technical Reference documentation (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details
+
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+from fivetran_connector_sdk import Logging as log
+import requests
+import json
+
+# Define base URL for Microsoft graph API
+base_url = "https://graph.microsoft.com/v1.0/"
```

Comment:
please use UPPER_CASE for constants
```suggestion
BASE_URL = "https://graph.microsoft.com/v1.0/"
```